datadog/tests/resource_datadog_integration_aws_event_bridge_test.go

```diff
@@ -0,0 +1,107 @@
+package test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/terraform"
+
+	"github.com/terraform-providers/terraform-provider-datadog/datadog/fwprovider"
+	"github.com/terraform-providers/terraform-provider-datadog/datadog/internal/utils"
+)
+
+func TestAccIntegrationAwsEventBridgeBasic(t *testing.T) {
+	t.Parallel()
+	ctx, providers, accProviders := testAccFrameworkMuxProviders(context.Background(), t)
+	accountID := uniqueAWSAccountID(ctx, t)
+	uniq := uniqueEntityName(ctx, t)
+
+	resource.Test(t, resource.TestCase{
+		ProtoV5ProviderFactories: accProviders,
+		CheckDestroy:             testAccCheckDatadogIntegrationAwsEventBridgeDestroy(providers.frameworkProvider),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckDatadogIntegrationAwsEventBridge(accountID, uniq),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckDatadogIntegrationAwsEventBridgeExists(providers.frameworkProvider),
+				),
+			},
+		},
+	})
+}
+
+func testAccCheckDatadogIntegrationAwsEventBridge(accountID string, uniq string) string {
+	return fmt.Sprintf(`
+	resource "datadog_integration_aws" "account" {
+		account_id                       = "%s"
+		role_name                        = "testacc-datadog-integration-role"
+	  }
+
+	resource "datadog_integration_aws_event_bridge" "foo" {
+		account_id = "%s"
+		event_generator_name = "%s"
```

Comment:
I wasn't able to get the tests/cassettes to pass when `create_event_bus = true`, I think because the mocked IAM role `testacc-datadog-integration-role` doesn't have the `events:CreateEventBus` permission, but I'm not totally sure. However, I tested that setting manually in my own sandbox org+sandbox aws and it worked.